eva_sub_cli/validators/validator.py

```diff
@@ -97,6 +103,20 @@ def report(self):
     def _validate(self):
         raise NotImplementedError
 
+    def set_up_output_dir(self):
+        if os.path.exists(self.output_dir):
+            backup_file_or_directory(self.output_dir, max_backups=9)
```

Comment:
That will work. My only concern is that we might pollute the user's directory structure:
```
user_provided_directory.1
user_provided_directory.2
user_provided_directory.3
user_provided_directory.4
...
```

Reply:
As discussed, this will only backup the validation output directory not the overall submission directory, so it should be fine.